modules/compute/gke-node-pool/variables.tf

```diff
@@ -319,3 +319,16 @@ variable "specific_reservation" {
     values = null
   }
 }
+
+variable "host_maintenance_interval" {
+  description = "Specifies the frequency of planned maintenance events."
+  type        = string
+  default     = "PERIODIC"
+  validation {
+    condition = var.host_maintenance_interval != null ? contains(
```

Comment:
Looking at this similar change https://github.com/GoogleCloudPlatform/cluster-toolkit/pull/2220 , we did not allow `null` as an allowed value. 

It would be great if we can stay consistent and use empty string as the "null" value.

Reply:
Makes sense, updated